charts/sdc/values.yaml

```diff
@@ -85,16 +97,29 @@ agents:
     ConnectionStrings__AgentsDatabase: "Host=postgresql; Database=agentsdb; Username=agentsadmin; Password=xxxxx; SearchPath=agents"
     DefaultApiClient__ClientId: "xxxxx"
```

Comment:
Why are ClientID and ClientSecret under database?

use generic names like clientId and clientSecret

Reply:
Instead of DefaultApiClient__ClientId and DefaultApiClient__ClientSecret